spec/std/colorize_spec.cr

```diff
@@ -17,116 +17,116 @@ private class ColorizeToS
   end
 end
 
-describe "colorize" do
+describe "colorize_wrap" do
```

Comment:
it's still about `colorize`, `colorize_wrap` is just an internal helper.

Reply:
It is failure of `%s/colorize/colorize_wrap`. Thank you.